freeipa-client/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -485,6 +485,11 @@ public Host showHost(String fqdn) throws FreeIpaClientException {
         try {
             RPCResponse<T> response = (RPCResponse<T>) jsonRpcHttpClient.invoke(method, List.of(flags, parameterMap), type);
             LOGGER.debug("Response object: {}", response);
+            if (response == null) {
```

Comment:
as a suggestion, you can start using checkState from guava to do this kind of thing in the future. It's more concise.

Reply:
noted. thanks!